core/modules/file/file.entity.inc

```diff
@@ -126,6 +128,58 @@ class File extends Entity {
       'options' => array(),
     );
   }
+
+  /**
+   * Overrides Entity::createAccess().
+   */
+  public static function createAccess($bundle = NULL, $account = NULL) {
+    // If no user object is supplied, the access check is for the current user.
+    if (empty($account)) {
+      $account = $GLOBALS['user'];
+    }
+
+    return user_access('access content', $account);
```

Comment:
Seems like this should always just return TRUE, as we don't have any requirements on this entity type. Since `Entity` defaults to returning TRUE, we can just delete this whole method.